js/test/date/testMonthTranslation.js

```diff
@@ -498,18 +498,18 @@ module.exports.testmonthtranslation = {
             value[i] = fmt.format(DateFactory({month:i+1, type:"gregorian"}));
         }
 
-        test.equal(value[0], "Ιανουάριος"); //standAlone
-        test.equal(value[1], "Φεβρουάριος");
-        test.equal(value[2], "Μάρτιος");
-        test.equal(value[3], "Απρίλιος");
-        test.equal(value[4], "Μάιος");
-        test.equal(value[5], "Ιούνιος");
-        test.equal(value[6], "Ιούλιος");
-        test.equal(value[7], "Αύγουστος");
-        test.equal(value[8], "Σεπτέμβριος");
-        test.equal(value[9], "Οκτώβριος");
-        test.equal(value[10], "Νοέμβριος");
-        test.equal(value[11], "Δεκέμβριος");
+        test.equal(value[0], "Ιανουαρίου"); //standAlone
```

Comment:
According to cldr 36, Modified values are wrong.
"stand-alone" ->
"wide": {
                  "1": "Ιανουάριος",
                  "2": "Φεβρουάριος",
                  "3": "Μάρτιος",
                  "4": "Απρίλιος",
                  "5": "Μάιος",
                  "6": "Ιούνιος",
                  "7": "Ιούλιος",
                  "8": "Αύγουστος",
                  "9": "Σεπτέμβριος",
                  "10": "Οκτώβριος",
                  "11": "Νοέμβριος",
                  "12": "Δεκέμβριος"
                }

Reply:
If you look in CLDR at the Greek date formats file, you will see that the availableFormats MMM and MMM use the regular month names instead of the stand-alone versions. If you look at the yMMM and yMMMM, that is where the stand-alone names are used. Previously, if the stand-alone month names are different than the regular month names, we used the stand-alone everywhere, which is not correct. With this change, the MMM and yMMM can be different.

In this case above, the format was MMMM, so the unit tests are now switched to the regular names instead of the stand-alone versions.

Reply:
oh.. If you so, My previous PR, https://github.com/iLib-js/iLib/pull/260 is also wrong.
Because I forced to switch from a character the `M` to `L` when the c20 length duration formatting.